recurly/__init__.py

```diff
@@ -1035,6 +1035,16 @@ def authorize(self):
         """
         return self.__invoice(self.collection_path + '/authorize')
 
+    def capture(self, transaction_uuid):
+        """
+        Allows the merchant to initiate a capture transaction tied to the original authorization.
+        transaction_uuid: The uuid for the transaction representing the authorization. Can typically be found at invoice_collection.charge_invoice.transactions[0].uuid.
+
+        Returns:
+            InvoiceCollection: The captured invoice collection
+        """
+        return self.__invoice(self.collection_path + '/transaction-uuid-' + transaction_uuid + '/capture')
```

Comment:
Does this work? I believe `__invoice` sends the purchase as a body but those endpoints don't take bodies (I don't think)

Reply:
@bhelx I drummed up some scripts to test it out. It _appears_ to work, as a transaction capture will appear in the UI, but I could definitely take a closer look and see if there's a more correct way to approach this.

Reply:
Does it send a body or not?

Reply:
I talked this over with @douglasmiller. Like you stated, `__invoice` does send the purchase as a body. `capture` and `cancel` shouldn't take bodies - and also, if it weren't enough of a hint, there isn't any body [documentation](https://dev.recurly.com/docs/capture-purchase) for those two endpoints. It would be better to model a different approach based on how `capture()` and `cancel()` are written in the [Ruby client library](https://github.com/recurly/recurly-client-ruby/blob/f99f8046cb615fcbea80741ffab54e1dce1fc4af/lib/recurly/purchase.rb#L205-L206).